tests/integration/adapter/find-all-test.js

```diff
@@ -244,44 +208,33 @@ test('isUpdating is true while records are fetched', function(assert) {
         },
       ],
     });
-  });
 
-  let persons = store.peekAll('person');
-  assert.equal(persons.get('length'), 1);
+    let persons = store.peekAll('person');
+    assert.equal(persons.get('length'), 1);
 
-  let wait = run(() => {
-    return store.findAll('person').then(persons => {
-      assert.equal(persons.get('isUpdating'), false);
-      assert.equal(persons.get('length'), 2);
+    let wait = run(() => {
+      return store.findAll('person').then(persons => {
+        assert.equal(persons.get('isUpdating'), false);
+        assert.equal(persons.get('length'), 2);
+      });
```

Comment:
A better pattern than:

```
let wait = run(() => return new Promise())
// ... assert intermediate state
await wait;
```

is

```
let promise = new Promise()
// ... assert intermediate state
await promise;
```

mostly this just means you can remove the `run` here. 👍